app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv" and command.split() == 3:
```

Comment:
swap and add len because it return list
```suggestion
    if len(command.split()) == 3 command.split()[0] == "mv":
```